templates/takeovers/_microk8s-takeover.html

```diff
@@ -0,0 +1,50 @@
+<section class="p-strip--image is-dark is-deep js-takeover p-takeover--microk8s">
+  <div class="row u-equal-heights">
```

Comment:
Shouldn't this be "u-equal-height"? (No s)

Reply:
Good spot :+1: